tests/evaluations_tests/test_eval_detection_voc.py

```diff
@@ -97,3 +98,57 @@ def test_eval_detection_voc_difficult(self):
             gt_difficults=gt_difficults, iou_thresh=self.iou_thresh)
         np.testing.assert_equal(results[0]['recall'], self.rec)
         np.testing.assert_equal(results[0]['precision'], self.prec)
+
+
+class TestEvalDetectionVOCConsistencyWithMATLAB(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        base_url = 'https://github.com/yuyu2172/' \
+                   'share-weights/releases/download/0.0.2'
+
+        cls.dataset = np.load(request.urlretrieve(os.path.join(
+            base_url, 'example_detection_dataset_2017_06_01.npz'))[0])
+        cls.result = np.load(request.urlretrieve(os.path.join(
+            base_url, 'example_detection_result_2017_06_01.npz'))[0])
```

Comment:
eval_detection_voc_result_*
How about?
We may add similar kind of file for COCO.